decidim-admin/app/views/decidim/admin/moderations/reports/show.html.erb

```diff
@@ -46,17 +47,18 @@
     <dt><%= t(".report_details") %></dt>
     <dd><%= @report.details %></dd>
   </dl>
+  </div>
+
   <div class="item__edit-sticky">
```

Comment:
I see the same indentation problem that I mentioned in another file here. Steps to reproduce:

vim  decidim-admin/app/views/decidim/admin/moderations/reports/show.html.erb

And then indenting the whole file with:
`=G` 

And saving it: 
`:wq`

And then `git diff`

Reply:
Leaving out indentation problems to a linter